deploy/cloudformation/manifest-pipeline-pipeline.yml

```diff
@@ -240,24 +199,10 @@ Resources:
       PolicyDocument:
         Statement:
           - Action:
-              - 's3:CreateBucket'
-              - 's3:DeleteBucket'
-              - 's3:PutBucketLogging'
-            Resource:
-              - 'arn:aws:s3:::mellon-manifest-pipeline*'
-            Effect: 'Allow'
-          - Action:
-              - 's3:GetObject'
-              - 's3:GetObjectVersion'
-              - 's3:GetBucketVersioning'
-            Resource:
-              - 'arn:aws:s3:::mellon-manifest-pipeline*/*'
-              - 'arn:aws:s3:::codepipeline*'
-            Effect: 'Allow'
-          - Action:
-              - 's3:PutObject'
-            Resource:
-              - 'arn:aws:s3:::codepipeline*'
+              - 'ssm:Get*'
```

Comment:
I think CodePipeline probably only needs `ssm:GetParameter*` if it's reading values; is there a scenario you're envisioning where the CD pipeline is creating or deleting parameters?